app/models/manageiq/providers/openshift/container_manager/refresher.rb

```diff
@@ -12,6 +12,16 @@ class ContainerManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
         {:name => 'images'}
       ]
 
+      def fetch_ems_metrics(ems)
+        hawk = ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::HawkularClient.new(ems)
```

Comment:
@yaacov here I am creating a new HawkularClient for the ems, even though the ems already has a hawkular client for metrics. This is done once in every refresh for each provider.
Is that a good practice? Do you see a better way to do it?